bpf/lib/common.h

```diff
@@ -79,25 +79,26 @@
 #define CILIUM_CALL_IPV6_TO_HOST_POLICY_ONLY	CILIUM_CALL_IPV6_TO_LXC_POLICY_ONLY
 #define CILIUM_CALL_IPV4_TO_ENDPOINT		13
 #define CILIUM_CALL_IPV6_TO_ENDPOINT		14
-#define CILIUM_CALL_IPV4_NODEPORT_NAT		15
-#define CILIUM_CALL_IPV6_NODEPORT_NAT		16
-#define CILIUM_CALL_IPV4_NODEPORT_REVNAT	17
-#define CILIUM_CALL_IPV6_NODEPORT_REVNAT	18
-#define CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT	19
-#define CILIUM_CALL_IPV4_NODEPORT_DSR		20
-#define CILIUM_CALL_IPV6_NODEPORT_DSR		21
-#define CILIUM_CALL_IPV4_FROM_HOST		22
-#define CILIUM_CALL_IPV6_FROM_HOST		23
-#define CILIUM_CALL_IPV6_ENCAP_NODEPORT_NAT	24
-#define CILIUM_CALL_IPV4_FROM_LXC_CONT		25
-#define CILIUM_CALL_IPV6_FROM_LXC_CONT		26
-#define CILIUM_CALL_IPV4_CT_INGRESS		27
-#define CILIUM_CALL_IPV4_CT_INGRESS_POLICY_ONLY	28
-#define CILIUM_CALL_IPV4_CT_EGRESS		29
-#define CILIUM_CALL_IPV6_CT_INGRESS		30
-#define CILIUM_CALL_IPV6_CT_INGRESS_POLICY_ONLY	31
-#define CILIUM_CALL_IPV6_CT_EGRESS		32
-#define CILIUM_CALL_SIZE			33
+#define CILIUM_CALL_IPV4_NODEPORT_NAT_EGRESS	15
+#define CILIUM_CALL_IPV4_NODEPORT_NAT_INGRESS	16
+#define CILIUM_CALL_IPV6_NODEPORT_NAT		17
```

Comment:
We need the similar new two calls for the IPv6 too.

Reply:
Yes I wanted to get your approval before to work on this part. I'm considering your are good with this approach! :-)